Bright Tides/Assets/Scripts/GameOverUIHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class GameOverUIHandler : MonoBehaviour {
+	public Text turnsText;
+	public Text goldText;
+	public Text ratingText;
+	public SceneState mainMenuSceneState;
+
+	// Use this for initialization
+	void Awake () {
+		int turns = GameManager.instance.GetTurnCount();
+		int gold = GameManager.instance.playerInstance.GetComponent<Entity>().attributes.gold;
```

Comment:
is there a way to make this show the total gold earned through the entire session? right now this would be total gold when the player dies, so they would not get score for purchased upgrades

Reply:
Not right now, good candidate for new enhancement issue though...